pennylane_lightning/core/src/simulators/lightning_gpu/utils/cuda_helpers.hpp

```diff
@@ -73,9 +73,10 @@ inline static auto Div(const CFP_t_T &a, const CFP_t_U &b) -> CFP_t_T {
  * @tparam CFP_t Complex data type. Supports std::complex<float>,
  * std::complex<double>, cuFloatComplex, cuDoubleComplex
  * @param a The given complex number
- * @return CFP_t The conjuagted complex number
+ * @return CFP_t The conjugated complex number
  */
-template <class CFP_t> inline static constexpr auto Conj(CFP_t a) -> CFP_t {
+template <class CFP_t>
+__device__ inline static constexpr auto Conj(CFP_t a) -> CFP_t {
```

Comment:
```suggestion
__host__ __device__ inline static constexpr auto Conj(CFP_t a) -> CFP_t {
```

Should we also make this host compatible?

Reply:
It seems it is not required at present, but we might as well.